src/Uno.UI.Composition/Composition/SkiaCompositionSurface.skia.cs

```diff
@@ -1,27 +1,39 @@
 ﻿#nullable enable
 
-
 using SkiaSharp;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Runtime.CompilerServices;
 using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 using Uno.Extensions;
 using Uno.Foundation.Logging;
+using Uno.UI.Dispatching;
 using Windows.Graphics;
 
 namespace Microsoft.UI.Composition
 {
-	internal partial class SkiaCompositionSurface : ICompositionSurface
+	internal partial class SkiaCompositionSurface : CompositionObject, ICompositionSurface
 	{
-		private SKImage? _image;
+		private SKImage[]? _images;
+		private SKCodecFrameInfo[]? _infos;
+		private int _currentFrame;
+		private int _totalDuration;
```

Comment:
Same for this, there is no need to keep it as a private field (less important as it's only a `int`, but sill better to keep variables in the smallest scope possible).